pkg/monitor/dynkeepalived.go

```diff
@@ -235,6 +263,10 @@ func KeepalivedWatch(kubeconfigPath, clusterConfigPath, templatePath, cfgPath st
 
 	go handleConfigModeUpdate(cfgPath, kubeconfigPath, updateModeCh)
 
+	if os.Getenv("IS_BOOTSTRAP") == "yes" {
```

Comment:
I really prefer the os.LookupEnv() method. It returns true or false based on existence. If it isn't set, it returns false. If it's set, it returns true. That way, we don't really care what the value is, just that it's set. IIUC, we only set IS_BOOTSTRAP on the bootstrap, right?

Reply:
Nope, we set value ("yes" for bootstrap and "no" for rest of the nodes) see [1], if we'll change the way we set these env variables it will make sense to use os.LookupEnv() . 

I'm not sure we can change this thing at this point (code freeze)  


[1] https://github.com/openshift/machine-config-operator/blob/master/templates/common/baremetal/files/baremetal-keepalived.yaml#L128-#129

Reply:
Fair enough.